### ISSUE TITLE
Can't find the necessary components for the vmware_fusion vagrant provider

### ISSUE THREAD

**Issue by moander** [GoogleCloudPlatform/kubernetes#9773]
I am following this guide on OS X:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/vagrant.md

I am able to run other vagrant's using the same provider.

```
$ ./cluster/kube-up.sh 
Starting cluster using provider: vagrant
... calling verify-prereqs
Can't find the necessary components for the vmware_fusion vagrant provider, please fix and retry.

**Comment by jameskyle** [comment]
Same here, looking at it.

**Comment by jameskyle** [comment]
It's likely because vmrun is not in your path. Patch incoming, but you can add /Applications/VMware\ Fusion.app/Contents/Library to your PATH as a workaround in the interim.

**Comment by moander** [comment]
Thanks @jameskyle. Modifying the path fixed the reported error, but there are still some problems. Looking at the following gist the minion fails to be created, but the master is created and started.

https://gist.github.com/moander/fb67060b39d2667066ab

```
/sbin/ifdown ens33

Stdout from the command:

ERROR    : [/etc/sysconfig/network-scripts/ifdown-eth] Device ens33 has MAC address 00:0C:29:29:F0:F0, instead of configured address 00:0C:29:16:5F:2B. Ignoring.
```

**Comment by stevesloka** [comment]
@moander I have the same issue as you at the moment. I bought the vmware adapter as I was unable to get the virtualbox vagrant cluster running. I'm going to be digging in today, will update if I find anything.

**Comment by stevesloka** [comment]
I found the same issue reported here #10906 @moander

**Comment by erictune** [comment]
I'm closing this as a duplicate of #10906.

**Comment by moander** [comment]
Please re-open this issue. It is not a duplicate of #10906.

**Comment by erictune** [comment]
reopened

**Comment by moander** [comment]
Resolved by #12067
